src/lib/common/utils.test.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from 'vitest';
+import * as utils from './utils';
+
+const TRUTHY_INPUT_DATA = ['TRUE', String(true), String(1)];
+for (const input of TRUTHY_INPUT_DATA) {
+  test(`correctly determines input value '${input}' as truthy`, () => {
+    expect(utils.determineTruthyString(input)).toBe(true);
+  });
+}
+
+const FALSY_INPUT_DATA = [
+  'FALSE',
+  '',
+  ``,
```

Comment:
Nit: This test case seems to be a duplicate.

Reply:
Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/455/commits/ec895ff296a0ba929d68b7864dacb287bc363b81>.